weave-js/src/core/suggest.test.ts

```diff
@@ -322,7 +322,7 @@ describe('autosuggest', () => {
 
     expectOnlyNodes(result);
     const autosuggestNames = suggestResultNames(result);
-    expect(autosuggestNames).toEqual(['x']);
+    expect(autosuggestNames).toContain('x');
```

Comment:
what is the new result here? These exact `toEqual` tests on suggestions have been helpful to no inadvertently add unexpected things. Curious the motivation of `toContain`

Reply:
This PR changes the suggestions to include the non-hidden root ops. So autosuggestNames becomes something like: `['x', 'project("", "")', 'range(, , )']`

It seemed like the intention of the test was that the variable is in the suggestions so I changed it to `toContain`. I'm expecting the suggested ops to change a bunch (more root ops, maybe better textual representations) and it seemed like this test would be a bit brittle.